ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml

```
@@ -1390,6 +1390,50 @@ tests:
       ASSISTED_CONFIG: |
         PLATFORM=external
     workflow: assisted-baremetal
+- as: e2e-metal-assisted-5-control-planes-4-17
```

```suggestion
- always_run: false
  as: e2e-metal-assisted-5-control-planes-4-17
```

I don't think we want to run these jobs each time a PR is opened?

---

Yes thank you !